### PR TITLE
api bump

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@php-coder @wozniakjan here's my attempt at this. we'll see which one makes it.

File: glide.yaml [hunk 1]
```diff
@@ -38,7 +38,7 @@ import:
 
 # openshift second
 - package: github.com/openshift/api
-  version: 0ce1df2db7debb15eddb25f3ae76df4180777221
+  version: master
```
Comment (bparees) [inline glide.yaml]: @liggitt @deads2k valid?

Comment (liggitt) [reply glide.yaml]: yes

**Comment by deads2k** [comment]
Are the original pulls for these already lgtm'd?

@openshift/sig-security

**Comment by liggitt** [comment]
glide, api, and generation changes LGTM. defer to build and security teams for the validation and use of the fields

**Comment by deads2k** [comment]
This is going to make the re-rebase to 1.11 GA much easier.  bumping queue priority.

**Comment by deads2k** [comment]
https://github.com/openshift/origin/issues/20157
/retest

**Comment by deads2k** [comment]
#20157 again 

/retest

**Comment by deads2k** [comment]
this is needed for the re-rebase to GA. conformance install was fine.  The base level is really close.  Nothing here appears to be gcp only.  merging.

**Comment by vrutkovs** [comment]
This PR seems to have broken the logic during upgrades in openshift-ansible - `oc adm policy reconcile-sccs --additive-only=true -o name` now prints changes to the stdout during 3.10.n to 3.10.n+1 upgrades.

Is it expected or a side-effect of this PR?

**Comment by liggitt** [comment]
I don't see anything in this PR that would have modified that. Are you passing --confirm?

**Comment by vrutkovs** [comment]
>I don't see anything in this PR that would have modified that

Weird, this error poped up approximately when this was merged.

>Are you passing --confirm?

No, isn't it ignored now? https://github.com/openshift/openshift-ansible/pull/8781 drops `--confirm` for storage.

I'll try adding `--confirm` flag

**Comment by liggitt** [comment]
> No, isn't it ignored now? openshift/openshift-ansible#8781 drops --confirm for storage.

that is a separate command. scc reconciliation still requires it.

**Comment by vrutkovs** [comment]
Adding `--confirm` didn't help - https://github.com/openshift/openshift-ansible/pull/9060.

In this test the upgrade is being run immediately after install, so its not the difference between minor versions. However I can't pinpoint the exact change which caused this, but the timing suggest its this PR

**Comment by sdodson** [comment]
The ansible code that's failing is intended to abort the upgrade if there have been changes to bootstrapped SCCs because if we were to reconcile them we'd stop on local customizations.

**Comment by liggitt** [comment]
I misunderstood the question. Yes, this PR added a new field to the SCCs that meant they needed reconciling. Why does that fail an upgrade?

**Comment by sdodson** [comment]
https://github.com/openshift/openshift-ansible/pull/8390 (and some subsequent cleanup) Added the check at the behest of support with feedback from @simo5
Current code https://github.com/openshift/openshift-ansible/blob/master/playbooks/openshift-master/private/upgrade.yml#L6-L27

**Comment by liggitt** [comment]
> openshift/openshift-ansible#8390 (and some subsequent cleanup) Added the check at the behest of support with feedback from @simo5
> Current code https://github.com/openshift/openshift-ansible/blob/master/playbooks/openshift-master/private/upgrade.yml#L6-L27

that appears to be checking SCC reconciliation with an oc version skewed from the API server, which will emit false positives around reconciliation for this new defaulted field

**Pull request merged**